sklearn/datasets/base.py

```diff
@@ -416,7 +419,8 @@ def load_breast_cancer(return_X_y=False):
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        full description of the dataset, 'filename' (added in version 0.20),
+        the physical location of breast cancer csv dataset.
```

Comment:
I am not in favor of  `(added in version 0.20)` in the narrative.

I would add a directive on the bottom of data

```python
        the physical location of iris csv dataset.

        .. versionadded:: 0.20
           The ``filename`` attribute.
```